android-o/AndroidCipher/README.md

```diff
@@ -0,0 +1,11 @@
+AndroidCipher
+=====
+
+AndroidCipher is a simple project which encrypts and decrypts string with usage of  android cipher.
+The reason of creating this project is to test new methods in android 8.1.
```

Comment:
I would maybe reword this line slightly, to instead say something like  "This sample demonstrates some of the new encryption algorithms available in Android 8.1"

Reply:
resolved